modules/tinymce/changelog.txt

```diff
@@ -5,8 +5,9 @@ Version 5.7.0 (TBD)
     Added the ability to resize `video` and `iframe` media elements #TINY-6229
     Added a new `font_css` setting for adding fonts to both the editor and the parent document #TINY-6199
     Added a new `ImageUploader` API to simplify uploading image data to the configured `images_upload_url` or `images_upload_handler` #TINY-4601
-    Added an Oxide variable to define the container background color in fullscreen mode #TINY-6903
-    Added a new `AfterProgressState` event that is fired after `editor.setProgressState` calls complete #TINY-6686
+    Added Oxide variable to define the container background color in fullscreen mode #TINY-6903
```

Comment:
This wasn't merged correctly, so some changes @tylerkelly13 did have been lost.

- [ ] Fix changelog merge issues

Reply:
I've fixed this, however @tylerkelly13 can you please review the new entry to make sure it looks okay?

Reply:
@lnewson,

Added an Oxide variable for defining....

?

Reply:
@tylerkelly13, it's multiple variables so that would be misleading.

Reply:
Added Oxide variables .....

Reply:
I'm confused? It already says that... are you looking at the above, as that's not what's currently in the changelog.